playground/nextjs-app-router/components/demo/Checkout.tsx

```diff
@@ -0,0 +1,95 @@
+import {
+  Checkout,
+  CheckoutButton,
+  CheckoutStatus,
+} from '@coinbase/onchainkit/checkout';
+import type { LifecycleStatus } from '@coinbase/onchainkit/checkout';
+import { useCallback, useMemo } from 'react';
+import { useContext } from 'react';
+import { AppContext, CheckoutTypes } from '../AppProvider';
+
+export default function CheckoutDemo() {
+  const { checkoutTypes, checkoutOptions } = useContext(AppContext);
+
+  const chargeIDKey = useMemo(() => {
+    return `${checkoutOptions?.chargeId}`;
+  }, [checkoutOptions]);
+
+  const productIDKey = useMemo(() => {
+    return `${checkoutOptions?.productId}`;
+  }, [checkoutOptions]);
+
+  const handleOnStatus = useCallback((status: LifecycleStatus) => {
+    console.log('Playground.Checkout.onStatus:', status);
+  }, []);
+
+  const createCharge = useCallback(async () => {
+    return Promise.resolve(checkoutOptions?.chargeId || '');
+  }, [checkoutOptions]);
+
+  const chargeIDDisabled = !checkoutOptions?.chargeId;
+  const productIDDisabled = !checkoutOptions?.productId;
+
+  return (
+    <div className="mx-auto grid w-1/2 gap-8">
+      {checkoutTypes === CheckoutTypes.ProductID && (
+        <>
+          {checkoutOptions?.productId ? (
+            <>
```

Comment:
unnecessary fragments

Reply:
typescript was complaining, I think anything within () requires a parent div

Reply:
yea, but Checkout should be your single parent